src/flow/nav/FlowStackRoutes.ts

```diff
@@ -10,6 +11,7 @@ import Step5CompleteScreen from '../ui/screens/Step5CompleteScreen';
 
 export const FLOW_STACK_SCREENS = Object.freeze({
   [HomeScreen.ROUTE]: HomeScreen,
+  [LoadingScreen.ROUTE]: LoadingScreen,
   [LoginScreen.ROUTE]: LoginScreen,
   [LogoutScreen.ROUTE]: LogoutScreen,
   [RegisterScreen.ROUTE]: RegisterScreen,
```

Comment:
[2/2]

Intermediate screen just before the app navigates into the intended entry point.